main.go

```diff
@@ -281,6 +281,14 @@ func runCLIApp(c *cli.Context) (err error) {
 				"Added environment http_proxy: %s\n",
 				p)
 		}
+		// Pass through the no_proxy enviroment variable.
```

Comment:
Similar in spirit to the https_proxy above, please call out in the comments as to when and why one would need this.